holisticai/explainability/metrics/feature_importance/local_importance/_lime_importance_spread.py

```diff
@@ -0,0 +1,78 @@
+import pandas as pd
+
+from holisticai.explainability.metrics.feature_importance.utils import importance_spread
+
+
+def feature_importance_spread_lime(
+    feature_importance, conditional_feature_importance, lime_importance
+):
+    if lime_importance == "dataset":
+        metric_name = "Dataset"
+        imp_spread = {
+            "Global": feature_importance.groupby("Sample Id").apply(
+                lambda df: importance_spread(
+                    df.set_index("Feature Label")["Importance"], divergence=True
+                )
+            )
+        }
+        for c, ccfi in conditional_feature_importance.items():
+            imp_spread.update(
+                {
+                    str(c): ccfi.groupby("Sample Id").apply(
+                        lambda df: importance_spread(
+                            df.set_index("Feature Label")["Importance"], divergence=True
+                        )
+                    )
+                }
+            )
+    else:
+        metric_name = "Features"
+        imp_spread = {
+            "Global": feature_importance.groupby("Feature Label")["Feature Rank"].apply(
+                lambda x: importance_spread(x, divergence=True)
+            )
+        }
+        for c, df_cls in conditional_feature_importance.items():
+            imp_spread[str(c)] = df_cls.groupby("Feature Label")["Feature Rank"].apply(
+                lambda x: importance_spread(x, divergence=True)
+            )
+
+    spread_imp_divergence = {
+        k: importance_spread(v, divergence=True) for k, v in imp_spread.items()
+    }
+    spread_imp_ratio = {
+        k: importance_spread(v, divergence=False) for k, v in imp_spread.items()
+    }
```

Comment:
imp spread comment 1/2 --> Here (line 44) and on line 41, you then calculate the importance_spread of these importance spread numbers, with both divergence = True and False.
And the metric below (line 46) is just the mean.

Am I understanding correctly? Would be keen to understand the intuition behind it.